packages/ui-components/src/components/AppShell/AppShell.component.tsx

```diff
@@ -36,12 +37,28 @@ export const AppShell = ({
     )
   }
 
+  const renderHeaderContainer = (
+    pageHeader?: AppShellProps["pageHeader"],
+    topNavigation?: AppShellProps["topNavigation"]
```

Comment:
**(Question/Non-Blocking Improvement)**

I see that this makes the type a little more reusable. 

However, we usually define types directly. One benefit is that the function is understandable to the developer without searching for the types. Makes things more readable.

Wdyt? @barsukov @MartinS-git

Reply:
That was my suggestion to reuse a types, kinda best practices in this case :)